src/ExpenserMain.java

```diff
@@ -4,6 +4,11 @@ public class ExpenserMain implements Expenser {
 	// Add Expense feature
 	@Override
 	public void addExpense (Expense Ex) {
+		Expense Exp = new Expense();
```

Comment:
Since the object can be created when input is gathered in the GUI, I'd recommend this would be used to add the object to the ArrayList in the User class.  Doing so, a method likely needs to be created to put an already created object into the list since it is private.   

Also, since the parameter is taking an already created expense object, creating a new one isn't useful.